src/crystal/system/win32/time.cr

```diff
@@ -1,9 +1,47 @@
+require "c/winbase"
+
 module Crystal::System::Time
+  # Win32 epoch is 1601-01-01 00:00:00 UTC
+  WINDOWS_EPOCH_IN_SECONDS = 50_491_123_200_i64
+
+  # Resolution of FILETIME is 100 nanoseconds
+  NANOSECONDS_PER_FILETIME_TICK = 100
+
+  NANOSECONDS_PER_SECOND    = 1_000_000_000
+  FILETIME_TICKS_PER_SECOND = NANOSECONDS_PER_SECOND / NANOSECONDS_PER_FILETIME_TICK
+
+  # TODO: For now, this method returns the UTC offset currently in place, ignoring *seconds*.
   def self.compute_utc_offset(seconds : Int64) : Int32
-    raise NotImplementedError.new("Crystal::System::Time.compute_utc_offset")
+    ret = LibC.GetTimeZoneInformation(out zone_information)
+    raise Errno.new("GetTimeZoneInformation") if ret == -1
```

Comment:
Looking at it again, shouldn't this be `WinError`?

Reply:
Oh, I didn't know this class existed... it's not used anywhere.

Reply:
Yes because windows has both `errno` and `GetLastError` for the C99 and win32 APIs respectively.